src/theme/components/TextFieldTheme.ts

```diff
@@ -6,14 +6,32 @@ export default function TextFieldTheme(): Components {
       styleOverrides: {
         root: {
           borderRadius: 6,
-          height: 28,
           paddingRight: 8,
         },
         input: {
           paddingLeft: 8,
+          paddingTop: 14.5,
```

Comment:
I think we should avoid fractions of pixels. Please ask @sandrasong for clarification on this component.